app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
+        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
need to add validations to check **vstr** dict shchema, because in case of the wrong data coming (f.e. there is no keys such as "expiration_date"), we'll get a lot of errors, like IndexError. need a mechanism here to handle such issues. this https://python-jsonschema.readthedocs.io/en/stable/ library may help